utils/parseHinfahrtRecon.ts

```diff
@@ -0,0 +1,87 @@
+import zlib from "node:zlib";
+import { z } from "zod/v4";
+
+export const parseHinfahrtReconCrude = (hinfahrtRecon: string) => {
+	const dirtyRegex = /A=1@O=.*?@a=\d+@/g;
+	const matches = hinfahrtRecon.match(dirtyRegex);
+
+	if (matches === null) {
```

Comment:
```suggestion
	if (matches == null) {
```
No strict equality needed here.

From mdn:

> [Return value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/>match#return_value)
>
>An [Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) whose >contents depend on the presence or absence of the global (g) flag, or [null](https://developer.mozilla.org/en-US/>docs/Web/JavaScript/Reference/Operators/null) if no matches are found.

Reply:
Strict equality should be preferred over weak equality so that developers don't need to memorize [this table](https://dorey.github.io/JavaScript-Equality-Table/).

Strict equality reduces mental load.